data-lineage/src/main/java/com/wipro/ats/bdre/lineage/entiity/Function.java

```diff
@@ -59,7 +64,9 @@ public Function(String label, List<String> inColumnNameList, String outAlias) {
 	public void setOutNode(Node outNode) {this.outNode = outNode;}
 	public List<String> getInColumnNameList() {return inColumnNameList;}
 	public void setInColumnNameList(List<String> inColumnNameList) {this.inColumnNameList = inColumnNameList;}
-	public String getOutAlias() {return outAlias;}
+	public String getOutAlias() {
+		return outAlias;
+	}
 	public void setOutAlias(String outAlias) {this.outAlias = outAlias;}
 
 	public static void updateFunctions(List<Table> inTableNodes, List<Table> outTableNodes, List<Column> inColumnNodes,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "updateFunctions" is 14 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "inTableNodes,outTableNodes,constants,relations". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1172)